localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -380,51 +397,27 @@ void EKFLocalizer::initEKF()
   ekf_.init(X, P, params_.extend_state_step);
 }
 
-/*
- * predictKinematicsModel
+/*  == Nonlinear model ==
```

Comment:
I think this model description should also be moved to somewhere else. (Maybe state_transition.cpp?)